source/includes/_introduction.md

```diff
@@ -3,33 +3,19 @@
 > To test your connection to our API please use
 
 ```shell
-curl https://cerego.com/api/echo 
-    -d '{"echo":"Test"}' 
-    -X POST 
-    -H "Content-Type: application/json"
+curl https://partners.cerego.com/v3/my/profile
+    -H "Authorization: Bearer <BEARER_TOKEN>"
```

Comment:
Since it looks like we're changing "Bearer token" to "API key" everywhere else, we might wanna do that here as well, and in the `_users.md` examples.

Reply:
This header is a fairly standard header that I wouldn't want to change https://swagger.io/docs/specification/authentication/bearer-authentication/ - I'd rather change the documentation to refer to it as a bearer token.

Reply:
The issue with calling it bearer token in the docs though I think is that bearer token is often associated with OAuth 2.0, and that might confuse a user of our API.

Reply:
This comment in the previous documentation was to try to address that confusion: https://github.com/ceregousa/slate/pull/12/files#diff-bddb86b88fe468024d2a0f9da274f7bbL17 - I feel like "API key" is a more generalized term that encompasses bearer token as well.

Reply:
I misread the original comment. Changing <BEARER_TOKEN> to <API_KEY> makes sense to me.